templates/desktop/organisations.html

```diff
@@ -4,210 +4,227 @@
 {% block meta_description %}Ubuntu Desktop combines enterprise-grade support, security and functionality with the best of open source.{% endblock %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1RuxCA6GYh9UriNv26HCZP9UHo4b4_9fYaaMtGQuH_xc/edit#{% endblock meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock %}
+
 {% block content %}
-<section class="p-strip--suru-bottomed">
-  <div class="row u-equal-height u-vertically-center">
-    <div class="col-6">
-      <h1>Bring Ubuntu to your <br> organisation</h1>
-      <p>
-        Ubuntu Desktop combines enterprise-grade support, security <br>and functionality with the best of open source.
-      </p>
-      <p><a class="p-button--positive js-invoke-modal" href="/desktop/contact-us">Contact us</a></p>
-      <p><a href="/pro">Secure open-source with Ubuntu Pro&nbsp;&rsaquo;</a></p>
-      <p><a class="p-button" href="https://assets.ubuntu.com/v1/a48fc9cd-Ubuntu%20Desktop%20DS%2021.06.23.pdf">Download the datasheet</a></p>
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section">
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="row">
+          <div class="col-medium-3">
+            <div class="p-section--shallow">
+              <h1>Bring Ubuntu to your organisation</h1>
+            </div>
+          </div>
+          <div class="col-medium-3">
+          <p>Ubuntu Desktop combines enterprise-grade support, security <br />and functionality with the best of open source.</p>
```

Comment:
```suggestion
          <p>Ubuntu Desktop combines enterprise-grade support, security <br class="u-hide--small u-hide--medium" />and functionality with the best of open source.</p>
```
It is looking funky on medium/small screens